utils/scapp.h

```diff
@@ -38,6 +38,18 @@ struct clampedOption
   T max;
 };
 
+ktx_pack_astc_block_size_e
+getAstcBlockSize(const char* block_size);
+
+ktx_pack_astc_quality_levels_e
+getAstcQualityLevel(const char* quality);
+
+ktx_pack_astc_encoder_function_e
+getAstcEncoderFunction(const char* function);
+
+ktx_pack_astc_encoder_mode_e
+getAstcEncoderMode(const char* mode);
+
 /*
 // Markdown doesn't work in files included by snipped{doc} or include{doc}
 // so the table below has to be laboriously done in html.
```

Comment:
This markdown also needs changing.

Reply:
Which markdown?

Reply:
Good idea to wait until we've settled on the language though I usually do it the other way round, writing the Doxygen comment first. In that, it's only the table and the definition list markers that need to be HTML. All the rest of the markup is Doxygen markup.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/f99d33c832d5dbbdf5c33d2f23c3b2135957e114